nemo/export/sentencepiece_tokenizer.py

```diff
@@ -29,11 +29,28 @@ class SentencePieceTokenizer:
         special_tokens: either list of special tokens or dictionary of token name to token value
         legacy: when set to True, the previous behavior of the SentecePiece wrapper will be restored,
             including the possibility to add special tokens inside wrapper.
+        tokenizer: wraps an existing tokenizer
     """
 
     def __init__(
-        self, model_path: str, special_tokens: Optional[Union[Dict[str, str], List[str]]] = None, legacy: bool = False
+        self,
+        model_path: Optional[str] = None,
+        special_tokens: Optional[Union[Dict[str, str], List[str]]] = None,
+        legacy: bool = False,
+        tokenizer: Optional[sentencepiece.SentencePieceProcessor] = None,
     ):
+        if tokenizer is not None:
+            self.tokenizer = tokenizer
```

Comment:
could this go into some auxiliary method as it's also kind of repeated in L59-70?